docs/whisper-data-import.md

```diff
@@ -0,0 +1,26 @@
+# Whisper data import
+
+Metrictank comes with a set of tools to import Whisper data into the chunk format used by Metrictank. 
+The usage of the tools is documented on [this page](https://github.com/grafana/metrictank/blob/master/docs/tools.md), this document specifically focuses on how the import procedure of Whisper files works.
+
+## Import procedure
+
+### Determination of metric names
+
+The importer utility walks recursively through the whisper directory, looking for files which end with `.wsp`. If there is a name filter defined it additionally filters all the discovered files by the given pattern. It then generates the metric name by removing the prefix defined via the `-whisper-directory` parameter from the file's path, removing the string `.wsp` suffix, and replacing all the `/` characters with `.`.
+
+For example if a file is located at `/opt/graphite/storage/my/metric/name.wsp` and the value given to `-whisper-directory` is `/opt/graphite/storage`, then the resulting metric name would be `my.metric.name`.
+
+Additionally there is an optional parameter called `-name-prefix` which allows the caller to prefix all metric names with a given string, which needs to include the tailing `.` if it is wanted. In the above example, if the value of `-name-prefix` is `my.prefix.` then the resulting metric name would by `my.prefix.my.metric.name`.
+
+### Schema conversion
+
+The whisper importer reader requires the user to provide the path to the storage schemas used at the import destination, it then reads and parses those schemas. During the import it iterates over all the whisper files that need to be imported and processes each of them as following:
```

Comment:
Let's make it very clear this is NOT the schemas file used locally on the graphite setup.